tests/dtm/test_rgw_s3_process_restart.py

```diff
@@ -291,3 +291,139 @@ def test_delete_during_rgw_svc_restart(self):
                       "was restarting")
 
         self.log.info("ENDED: Verify DELETEs during rgw_s3 service restart using pkill")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-42250")
+    def test_write_read_delete_during_rgw_restart(self):
+        """Verify WRITE, READ, DELETE during rgw restart using pkill."""
+        self.log.info("STARTED: Verify WRITE, READ, DELETE during rgw restart using pkill")
+        bucket_name = 'bucket-test-42250'
+        object_prefix = 'object-test-42250'
+        log_file_prefix = 'test-42250'
+
+        que = multiprocessing.Queue()
+
+        self.log.info("Step 1: Start Write Operations for parallel reads "
```

Comment:
so everything will be run in single bucket?

Reply:
No it will on different buckets.